README.md

```diff
@@ -53,6 +53,7 @@ The available configs are:
 - [No Implicit Buggy Globals](./docs/rules/no-implicit-buggy-globals.md)
 - [No Inner HTML](./docs/rules/no-inner-html.md)
 - [No InnerText](./docs/rules/no-innerText.md)
+- [No Script Tag Building](./docs/rules/no-script-tag-building.md)
```

Comment:
🚲 🏠 Def a bike shed so disregard at your discretion, but maybe this should be called **"No Dynamic Script Tags"**? I notice that this is in fact what you refer to them as in all the description text in this PR for example.